extensions/amp-base-carousel/0.1/amp-base-carousel.css

```diff
@@ -71,25 +71,134 @@ amp-base-carousel[i-amphtml-base-carousel-hide-buttons] .i-amphtml-base-carousel
   pointer-events: none;
 }
 
-.i-amphtml-base-carousel-prev,
-.i-amphtml-base-carousel-next {
-  filter: drop-shadow(0px 1px 2px #4a4a4a);
-  width: 40px;
-  height: 40px;
-  padding: 20px;
+.i-amphtml-base-carousel-arrow-backdrop,
+.i-amphtml-base-carousel-arrow-background,
+.i-amphtml-base-carousel-arrow-frosting {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+}
+
+
+.i-amphtml-base-carousel-arrow-frosting {
+  /**
+   * Add a frosted glass effect to the pagination dots background. The backdrop
+   * and background elements will apply a darkening on top of the blurring.
+   */
+  backdrop-filter: blur(3px);
+}
+
+.i-amphtml-base-carousel-arrow-backdrop {
+  /**
+   * Instead of using a plain background color, we use a backdrop-filter to
+   * make the effective background partially dependent on the current
+   * backdrop (e.g. the image on a carousel slide). This allows the pagination
+   * backgroud to be distinguishable on very dark backgrounds, while still
+   * having it be semi-transparent.
+   *
+   * This can be harder to distinguish on middle greys. For this current
+   * current configuration, hsl(0, 0%, 31%) to hsl(0, 0%, 35%) can appear quite
+   * close to the background color.
+   *
+   * The blur here is used for averaging the backdrop color, so that sharp
+   * contrasts (e.g. edge of sky and landscape) do not cause immediate shifts
+   * in the container color.
+   *
+   * The greyscale is used to have the container slightly tinted by the inverse
+   * color, but also not be too tinted.
+   */
+   backdrop-filter: blur(12px) invert(1) grayscale(0.6) brightness(0.8);
```

Comment:
did we test what this looks like on browsers that don't support `backdrop-filter` yet? https://caniuse.com/#feat=css-backdrop-filter

Reply:
Yep, it still gets a background of `rgba(0, 0, 0, 0.3)` from another rule, which is fine but could be a bit better. I didn't feel it was bad enough to need to use an `@supports` rule instead since the vast majority of browsers in use support `backdrop-filter`.